### PR TITLE
Lxc 186

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
For testing this you can do:
```
liferay cfg set docker.localdev.server.pull-image true
liferay cfg set docker.localdev.server.image rotty3000/localdev-server:v20221116-15-g0807e0b
```

File: docker/images.go [hunk 1]
```diff
@@ -1,36 +1,101 @@
...
 	"liferay.com/liferay/cli/constants"
+	lstrings "liferay.com/liferay/cli/strings"
 )
 
 func BuildImages(verbose bool) {
 	var s *spinner.Spinner
 
+	image := viper.GetString(constants.Const.DockerLocaldevServerImage)
+
+	if viper.GetBool(constants.Const.DockerLocaldevServerPullimage) {
```
Comment (gamerson) [inline docker/images.go:28]: I'm assuming that eventually we want to provide some type of migration or on-ramp to this setting.  When I'm trying this out and I will have to manually set both image and pullImage configs to their new values.  Existing users will still be pointing to localdev local image.

**Review by gamerson — APPROVED** [review]
LGTM!  Hey @rotty3000  this worked for me in my testing.  As long as we have a new default for brand new people I'm good.  Also thinking about existing users, how to migrate them.  But that could be a followon PR.

**Comment by rotty3000** [comment]
yeah, migration is something we'll have to plan out. Maybe we can check the stored value against the original and we'd know if it had changed or not and we can then set a new default to the official image... or something.

**Comment by gamerson** [comment]
I'm good to merge this.

**Pull request merged**